cmd/spire-server/cli/run/run.go

```diff
@@ -668,6 +670,7 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 	}
 
 	sc.AuthOpaPolicyEngineConfig = c.Server.Experimental.AuthOpaPolicyEngine
+	sc.ExcludeSNFromCASubject = c.Server.Experimental.ExcludeSNFromCASubject
```

Comment:
Since we'd like to make this the default behavior, and are only including this temporary flag for one minor release for backcompat, I'd recommend adding some logging when this is set. Something like:
```
// TODO: remove exclude_sn_from_ca_subject in SPIRE v1.10.0
if sc.ExcludeSNFromCASubject {
	sc.Log.Warn("The deprecated exclude_sn_from_ca_subject configurable will be removed in a future release")
}
```